tests/std/tests/GH_002030_asan_annotate_string/env.lst

```diff
@@ -2,9 +2,9 @@
 # SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 
 # This test matrix is the usual test matrix, with all currently unsupported options removed, crossed with the ASan flags.
-# TRANSITION, VSO-1350252
-# Due to a bug in the ASan libs using ASan with /MD or /MT requires IDL==0 and using /MDd or /MTd requires IDL==2.
-# clang-cl does not currently support targeting /MDd or /MTd.
+
+# TRANSITION, VSO-1350252 - due to vcasan.lib including the standard library, we can't use it (pending new release).

```

Comment:
Which release, specifically? This will help us notice when the workaround should be removed during a toolset update. Also applies to `GH_002030_asan_annotate_vector/env.lst`.